apps/src/lib/metrics/DashboardMetricsApi.ts

```diff
@@ -0,0 +1,29 @@
+import {
+  getAuthenticityToken,
+  AUTHENTICITY_TOKEN_HEADER,
+} from '../../util/AuthenticityTokenStore';
+import {MetricsApi} from './MetricsApi';
+
+const BASE_URL = '/browser_events/';
+
+/**
+ * A {@link MetricsApi} implementation that forwards metrics to Dashboard.
+ */
+export default class DashboardMetricsApi implements MetricsApi {
+  async sendLogs(logs: object[]): Promise<Response> {
+    let token;
+    try {
+      token = await getAuthenticityToken();
+    } catch (error) {
+      return Response.error();
+    }
+
+    return fetch(BASE_URL + 'put_logs', {
+      method: 'POST',
+      body: JSON.stringify({logs}),
+      headers: {
+        [AUTHENTICITY_TOKEN_HEADER]: token,
+      },
+    });
+  }
+}
```

Comment:
Checking my understanding: Is this basically providing a way for us to send log data to CloudWatch with an authenticity token?

Reply:
Basically yes; this proxies the metrics call through Dashboard, which in turn posts it to cloudwatch. The Dashboard API that we're hitting requires an authenticity token, which is why we need to grab it. Technically on the client side, we don't know that this is eventually going to Cloudwatch, we just know that we're sending it over to dashboard to handle.